frontend/src/components/Item/ItemMenu.vue

```diff
@@ -244,10 +244,10 @@ function getLibraryOptions(): MenuOption[] {
             useSnackbar(t('libraryRefreshQueued'), 'normal');
             taskManager.startTask({
               type: TaskType.LibraryRefresh,
-              id: menuProps.item.Id,
-              data: menuProps.item.Name,
+              id: menuProps.item.Id as string,
+              data: menuProps.item.Name as string,
```

Comment:
These casts are really just not-null assertions. I figured that was better than casting the entire object below. These can be more easily cleared out later I think.